# Goal
float64[] end_position
---
# Result
bool success
---
# Feedback
float64[] position_error